docs/customize_look&feel_guide.md

```diff
@@ -0,0 +1,39 @@
+# Customize the look & feel of Harbor
+
+The primary look & feel of Harbor supports to be customized with several simple steps. All the relevant customization configurations are saved in the `setting.json` file under `$HARBOR_DIR/src/ui_ng/src` folder with `json` format and will be loaded when the application is started to run. 
+
+## Configure
+Open the `setting.json` file, you'll see the default content as shown below:
+
+```
+{
+  "headerBgColor": "#004a70",
+  "headerLogo": "",
+  "loginBgImg": "",
+  "product": {
+    "title": "VMware Harbor",
+    "company": "vmware",
+    "name": "Harbor",
+    "introductions": {
+      "zh-cn": "",
+      "es-es": "",
+      "en-us": ""
+    }
+  }
+}
+```
+
+Change the configuration value if you want to override the default style to your own. Here are references:
```

Comment:
Change the configuration value if you want to override the default style to your own. Here are references:
==> Change the values of configuration if you want to override the default style to your own. Here are references: